Makefile

```
@@ -1,9 +1,6 @@
 #!/usr/bin/make -f
 
-all: build
-
-rosetta:
-	go build -mod=readonly ./cmd/rosetta
+all: build plugin test test-rosetta-ci
```

minor thing, `all` should:
```suggestion
all: build plugin
```